### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by klay2000**

### CODE REVIEW

**Review by rtasson — APPROVED** [review]

**Comment by Lohrer** [comment]
How was anything working before if all that stuff was missing from `web/static`? Is it really all required?

**Review by linknum23 — APPROVED** [review]

File: web/templates/rest-api-doc.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <script type="module" src="/static/js/libs/rapidoc/9.3.3/rapidoc-min.js"></script>
+    <rapi-doc spec-url="/openapi.yaml" regular-font="Open Sans" mono-font="Roboto Mono" render-style="read" show-header="false" theme="dark"  header-color="#47AFE8" primary-color="#47AFE8" allow-authentication="false">
+      <!-- NOTE: slot="" is used to put inner html where you want it" -->
+      <!-- Image -->
+      <img slot="nav-logo" src="/static/imgs/amplipi_icon_black.png"
+           style="width:90%; display: block; margin-left: auto;margin-right: auto">
+      <!-- content at the top -->
+      <!-- content at the bottom -->
+      <p slot="footer" style="margin:0; padding:16px 36px; background-color:#47AFE8; color:#fff; text-align:center;">
+        © Micronova 2022
```
Comment (Lohrer) [inline web/templates/rest-api-doc.html:31]: It's not 2022 anymore

**Review by Lohrer — APPROVED** [review]
Put the correct year in rest-api-doc.html and this looks good to me. I don't actually have any clue on the new web interface setup so I can't comment from that perspective...

File: web/templates/rest-api-doc.html (hunk 1 above)
Comment (klay2000) [reply web/templates/rest-api-doc.html:31]: is there a reason we have to hard code the year like that?

Comment (Lohrer) [reply web/templates/rest-api-doc.html]: I don't know, you added this file but did it come from somewhere else?

Comment (klay2000) [reply web/templates/rest-api-doc.html]: yeah, it's part of the api docs, they got ripped out by someone at some point

**Comment by Lohrer** [comment]
@rtertiaer, next time rebase before merging please!

**Pull request merged**